applications/ChimeraApplication/custom_strategies/custom_builder_and_solver/residualbased_block_builder_and_solver_with_constraints_for_chimera.h

```diff
@@ -68,7 +68,7 @@ namespace Kratos
 template <class TSparseSpace,
           class TDenseSpace,
           class TLinearSolver>
-class KRATOS_API(CHIMERA_APPLICATION) ResidualBasedBlockBuilderAndSolverWithConstraintsForChimera
+class ResidualBasedBlockBuilderAndSolverWithConstraintsForChimera
```

Comment:
why should'nt it be in the CHIM_APP ? if it is in Chimm app it should be no ?

Reply:
see my commit message, `KRATOS_API` must only be used with header-source separated files
if you use it in a header-only file then it fails